db/migrate/20150512083842_remove_old_translation_from_listing_unit.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOldTranslationFromListingUnit < ActiveRecord::Migration
+  def up
+    remove_column :listing_units, :translation_key
+  end
+
+  def down
+    add_column :listing_units, :translation_key, :string, limit: 64, after: :quantity_selector
+  end
+end
```

Comment:
Yep, that will do it.

Also, when you deploy the code, the deploy script asks which migrations to run. DO NOT RUN THIS MIGRATION. Wait for the 2 minutes for the new code to kick in, and after that run this migration manually.